### DISCUSSION TITLE
Open DBeaver PostgreSQL connection via CLI

### DISCUSSION THREAD

**Discussion by thedomdom** [dbeaver/dbeaver#20901] [Q&A] [answered]
### Description

Dear DBeaver community,

is it possible to open a DBeaver PostgreSQL connection via the CLI?

I installed DBeaver via `brew` (`brew install --cask dbeaver-community`) and put the `dbeaver` executable into the `PATH` (`export PATH="/Applications/DBeaver.app/Contents/MacOS:$PATH"`).

I am now trying to open a DBeaver PostgreSQL connection via the CLI. After reading the [CLI documentation][1], I tried the following approaches:

- `dbeaver -con driver=postgresql|url=localhost:4444/giGgaPlyUPwc`
- `dbeaver -con driver=postgresql|host=localhost|port=4444|database=giGgaPlyUPwc|user=user|password=secret`

However, I always get the following warning in the console:

```
2023-06-14 14:25:04.811 - Initiate connection to PostgreSQL database [giGgaPlyUPwc@localhost] for Main <giGgaPlyUPwc>
2023-06-14 14:25:04.836 - {host}
java.net.UnknownHostException: {host}
```

Do you know why `dbeaver` tries to connect to the host `host` instead of `localhost`? Is this a bug or am I just using the CLI incorrectly?

Best regards!

  [1]: https://github.com/dbeaver/dbeaver/wiki/Command-Line

### DBeaver Version

dbeaver-ce-23.1.0-macos-aarch64.dmg

### Operating System

macOS Ventura Version 13.4

### Database and driver

Database: PostgreSQL
Driver: PostgreSQL 42.5.2 (not sure here... there is a file named `~/Library/DBeaverData/drivers/maven/maven-central/org.postgresql/postgresql-42.5.2.jar`)

### Steps to reproduce

See above:

- `dbeaver -con driver=postgresql|url=localhost:4444/giGgaPlyUPwc`
- `dbeaver -con driver=postgresql|host=localhost|port=4444|database=giGgaPlyUPwc|user=user|password=secret`

### Additional context

local database running, setting up the connection manually works.

**Comment by E1izabeth** [comment, marked as answer]
Have you tried to put connection parameters into quotes?

**Comment by E1izabeth** [comment]
It's been a while since no update here. If the issue is still actual please let me know and provide additional information.

**Comment by thedomdom** [comment]
I did not have the need for DBeaver in the past month. But tried it out today with quotes – works like a charm.
Thank you :)
